core/src/test/java/com/google/zxing/oned/Code128WriterTestCase.java

```diff
@@ -90,6 +108,10 @@ public void testEncodeWithFunc4() throws WriterException {
     assertEquals(expected, actual);
   }
 
+  private static BitArray matrixToArray(BitMatrix result) {
```

Comment:
This is just used one place, and it doesn't really make something into an array, just returns one row right?